posts/2025-01-09-Rust-1.84.0.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: "Announcing Rust 1.84.0"
+author: The Rust Release Team
+release: true
+---
+
+The Rust team is happy to announce a new version of Rust, 1.84.0. Rust is a programming language empowering everyone to build reliable and efficient software.
+
+If you have a previous version of Rust installed via `rustup`, you can get 1.84.0 with:
+
+```console
+$ rustup update stable
+```
+
+If you don't have it already, you can [get `rustup`](https://www.rust-lang.org/install.html) from the appropriate page on our website, and check out the [detailed release notes for 1.84.0](https://doc.rust-lang.org/stable/releases.html#version-1840-2025-01-09).
+
+If you'd like to help us out by testing future releases, you might consider updating locally to use the beta channel (`rustup default beta`) or the nightly channel (`rustup default nightly`). Please [report](https://github.com/rust-lang/rust/issues/new/choose) any bugs you might come across!
+
+## What's in 1.84.0 stable
+
+### Cargo can use toolchain version for library version selection
+
+1.84.0 stabilizes the minimum supported Rust version (MSRV) aware resolver,
+which uses the declared [minimum supported Rust version](https://doc.rust-lang.org/cargo/reference/rust-version.html) from
+dependencies, if available, to improve package version selection. Rust version
```

Comment:
We've found some people assuming the MSRV-aware resolver will respect your current Rust version and not your `package.rust-version`.  Instead your toolchain version is only a fallback for when your you don't set a `package.rust-version`.

```suggestion
which prefers dependency versions that have declared a [MSRV](https://doc.rust-lang.org/cargo/reference/rust-version.html) that is compatible with your project's MSRV. Rust version
```
(switched the link to use the acronym since it is now introduced earlier)

Similarly, I wonder if the title will encourage this misunderstanding but don't have much of a suggestion.